utils/validation/authSchemas.js

```diff
@@ -0,0 +1,17 @@
+const Joi = require("joi");
+
+const registerSchema = Joi.object({
```

Comment:
```suggestion
const registerValidationSchema = Joi.object({
```
аналогічно для інших